geth/jail/internal/timers/timers.go

```diff
@@ -9,62 +9,83 @@ import (
 	"github.com/status-im/status-go/geth/jail/internal/vm"
 )
 
-var minDelay = map[bool]int64{
-	true:  10,
-	false: 4,
-}
+type intervalFunc func(*loop.Loop) func(call otto.FunctionCall) otto.Value
 
-//Define jail timers
+// Define jail timers
 func Define(vm *vm.VM, l *loop.Loop) error {
+	var err error
```

Comment:
It's almost what I meant :D I think it should be possible to get rid of this declaration and put `return nil` at the end of this function and just use `:=` as `err` does not need to leak to the upper scope from any `if` statement.

Reply:
That's been fixed now :)